config.json

```diff
@@ -217,7 +217,8 @@
         "prerequisites": [
           "classes",
           "strings",
-          "if-else-statements"
+          "if-else-statements",
+          "ternary-operators"
```

Comment:
I'm not sure we should add `ternary-operators` as prerequisite. Although it's possible to use one, it's not like the exercise can't be solved without using ternary expressions.

In general I think we should only add prerequisites if the concepts are really required in order to solve the exercise.